### PR TITLE
Manage Students Table - add success / failure notifications

### PR DESCRIPTION
**Pull request by caleybrock**
screen shot 2018-02-23 at 3 52 17 pm
screen shot 2018-02-23 at 3 47 29 pm
One of these appear at the top of the table after adding a student.

### CODE REVIEW

File: apps/src/templates/manageStudents/ManageStudentsTable.jsx [hunk 1]
```diff
@@ -92,6 +93,7 @@ class ManageStudentsTable extends Component {
     studentData: PropTypes.arrayOf(studentSectionDataPropType),
     loginType: PropTypes.string,
     editingData: PropTypes.object,
+    addStatus: PropTypes.string,
```
Comment (islemaster) [inline apps/src/templates/manageStudents/ManageStudentsTable.jsx]:
```jsx
PropTypes.oneOf(Object.values(ADD_STATUS))
```

**Comment by caleybrock** [comment]
There should be test coverage for all redux actions now.

File: apps/src/templates/Notification.jsx [hunk 2]
```diff
@@ -119,7 +119,8 @@ class Notification extends Component {
     dismissible: PropTypes.bool.isRequired,
     newWindow: PropTypes.bool,
     analyticId: PropTypes.string,
-    isRtl: PropTypes.bool.isRequired,
+    // isRtl not required so it can be used on Pegasus
+    isRtl: PropTypes.bool,
```
Comment (Erin007) [inline apps/src/templates/Notification.jsx]: YourSchool gets isRtl from Redux and it's on Pegasus.  Can we do something similar: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/sites/code.org/pages/public/yourschool.js#L7

Comment (caleybrock) [reply apps/src/templates/Notification.jsx]: well that was easy to fix. thanks for the link.

**Pull request merged**